### DISCUSSION TITLE
REI not showing any items in PaperMC 1.20.1 (115) server.

### DISCUSSION THREAD

**Discussion by daltonikiis** [shedaniel/RoughlyEnoughItems#1473] [Q & A]
Not sure when this happened but it did happen sometime whilst I was playing and wasn't looking in my inventory.
![image](https://github.com/shedaniel/RoughlyEnoughItems/assets/141339245/b8f65793-9b55-4bf0-8b92-ef7d46745ce8)
![image](https://github.com/shedaniel/RoughlyEnoughItems/assets/141339245/a33271d3-80b9-4437-acf0-2e93c3ef9a68)
I don't have any mods installed that add extra items or interfere with any other mods for that matter, however in a survival single player world everything works fine. Any ideas? If required I play on Fabric 1.20.1 (0.14.22)

**Comment by wusel95** [comment]
I have the same issue since today suprisingly.... I temporary fix it by opening Inventoy, klick on the Settings button > go to the 4th Tab "Advanced" (i guess its Advanced in english, i have it in german) > and then click "Reload Plugins". After this the items are visible for the session im on the server.
